dashboard/app/models/game.rb

```diff
@@ -202,7 +208,7 @@ def uses_pusher?
   end
 
   def uses_small_footer?
-    [NETSIM, APPLAB, TEXT_COMPRESSION, GAMELAB, WEBLAB, SCRATCH, DANCE].include? app
+    [NETSIM, APPLAB, TEXT_COMPRESSION, GAMELAB, WEBLAB, SCRATCH, DANCE, SPRITELAB].include? app
```

Comment:
I think we want to exclude Sprite Lab, here so we get the language dropdown for Sprite Lab levels.